src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import Header from './components/Header'
+import Footer from './components/Footer'
+import Home from './pages/Home'
+import CatIndex from './pages/CatIndex'
+import CatShow from './pages/CatShow'
+import CatNew from './pages/CatNew'
+import CatEdit from './pages/CatEdit'
+import NotFound from './pages/NotFound'
+import cats from './mockCats.js'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      cats: cats
+    }
+  }
+  render(){
+    console.log(this.state.cats)
+    return(
+      <>
+        <h1>Home is where my cat is!</h1>
+        <Header />
+        <Router>
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route path="/catindex" component={CatIndex} />
+            <Route path="/catshow" component={CatShow} />
+            <Route path="/catnew" component={CatNew} />
+            <Route path="/catedit" component={CatEdit} />
+            <Route component={NotFound}/>
+          </Switch>
+        </Router>
```

Comment:
Looks good but Router should be the top layer App.js JSX tag. It will wrap your header, h1 and switch just like you have them